jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  roots: [
+    '<rootDir>/src',
+    '<rootDir>/tests'
+  ],
+  setupFiles: [
+    'jest-plugin-context/setup'
+  ],
+  moduleFileExtensions: ['js', 'ts'],
+  testResultsProcessor: 'jest-sonar-reporter',
+  testMatch: ['**/*.*Test.ts']
```

Comment:
Don't you have to add `  transform: {
    "^.+\\.ts$": "ts-jest",
  },` to compile typescript?

Reply:
Ok, thanks for explaining that this part is handled by `preset: 'ts-jest',`